bin/marked

```diff
@@ -13,7 +13,7 @@ var fs = require('fs'),
  * Man Page
  */
 
-function help() {
+function help () {
```

Comment:
But why spaces?

Reply:
I figured it should be consistant and since we are using standard might as well use their recommendation.

I personally don't care one way or the other but they were inconsistent before.

Reply:
I think it should have the fewest changes possible (preferably none) to `lib/marked.js` and `bin/marked`

Reply:
As for the philosophical question "why?" I think the argument is that is makes visual grepping easier.

`help()` = calling a help function
`help ()` = defining a help function

Reply:
I think I did pick the least changes. Making it consistent would have changed it either way.

Reply:
My brain takes issue with this. Which standard/style/recommendation is this from (sorry, might have missed a mention somewhere)?

I haven't seen a language or style that adds a space between the name and the argument list container. Mainly just curious because it seems like it might add confusion not reduce it.

Reply:
> I think I did pick the least changes

This is changing every single function. This is not necessary. We should make the tools follow the standards of the project, not conform to someone else's standard (unless there is clear value added).

Reply:
[standard js](https://standardjs.com/rules.html#javascript-standard-style) (which we [have been using](https://github.com/markedjs/marked/commit/2fa274284ea7e237608b947f3be03918eb31b5f1#diff-df39304d828831c44a2b9f38cd45289c) with eslint) has [space-before-function-paren](https://eslint.org/docs/rules/space-before-function-paren) eslint rule set to "always". We had that rule turned off which made our code inconsistant.

This PR removes `standard` from eslint since it doesn't allow us to catch es6 syntax in eslint but I added all of standard's rules to keep it consistant.

I changed it to never allow a space between a function and parens

Reply:
Thanks for the link.